docs/examples/demoscenario.md

```diff
@@ -0,0 +1,96 @@
+# Usage of GitHub Connector <!-- omit in toc -->
+
+- [Prerequisites](#prerequisites)
+- [Configuration of ServiceInstance to use Lambda functions](#configuration-of-serviceinstance-to-use-lambda-functions)
+	- [Option 1. Through Kyma user interface](#option-1-through-kyma-user-interface)
+		- [Binding application to the namespace](#binding-application-to-the-namespace)
+		- [Setting up ServiceInstance and Lambda](#setting-up-serviceinstance-and-lambda)
+		- [Creation of Lambda function](#creation-of-lambda-function)
+	- [Option 2. Shell script](#option-2-shell-script)
+
+## Prerequisites
+
+- Kyma with GitHub Connector deployed on your cluster (see [installation guide](helm-installation-tutorial.md)).
+- WebHook configured to deliver payload to the Connector's ```/webhook``` endpoint.
+
+## Configuration of ServiceInstance to use Lambda functions
+
+To write lambda in Kyma that can utilise the functionalities of GitHub Connector you must configure the application binding and ServiceInstance.
+
+### Option 1. Through Kyma user interface
```

Comment:
I know that you must've spent quite some time describing this flow and capturing screenshots but I suggest that you get rid of this section altogether. In Kyma docs the CLI is a first-class citizen, meaning that we document getting from point a to point b only through the terminal. There are exceptions of course as a few components and aspects of Kyma can be adjusted only through the UI. 

Furthermore, why would I want to use this flow instead of simply running a single script you guys prepared? ;)

Reply:
We decided to keep that because: firstly - one may have troubles running this script on Windows, secondly - the graphics tutorial shows user what is actually created during configuration process and they can adjust the process to their own needs.

Reply:
https://kyma-project.io/community/guidelines/content/#content-strategy-content-strategy-the-main-purpose-of-instructions

> The main purpose of instructions
One of the main Kyma principles is that CLI is, metaphorically speaking, the first-level citizen. Therefore, the documentation's main focus is to explain concepts and provide step-by-step instructions using CLI commands instead of Console UI screenshots. The only exception from the rule applies to the components that cannot be managed through the CLI, in which case screenshots are essential to explain specific functionalities.

Reply:
Your screenshots don't follow the guidelines either: 

https://kyma-project.io/community/guidelines/content/#screenshots-screenshots

Reply:
If you want to explain what your script does you can describe it here - this approach has been done in our docs on numerous occasions. 
As for customization options - you can use comments in your script. Describe the tweakable parameters in the doc, link to specific lines in the .sh file.

Reply:
The fact that I have to jump between your doc and the UI to see what actions should I perform is a nuisance and is likely to scare people away rather than attract them to trying your connector.

Reply:
We removed this guide.